era_5g_relay_network_application/worker_socketio.py

```diff
@@ -3,6 +3,8 @@
 from threading import Event, Thread
 from typing import Any, Optional
 
+from socketio.exceptions import BadNamespaceError  # pants: no-infer-dep
```

Comment:
`# pants: no-infer-dep` should not be required here - that's for ROS-related stuff, which has to be pre-installed.

Reply:
ok, removed (Copilot recommended it to me, and who am I to question their suggestions :))